src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/service/LocationService.kt

```diff
@@ -635,7 +635,7 @@ class LocationService(
     return (
       startLocation?.findAllLeafLocations() ?: cellLocationRepository.findAllByPrisonIdAndActive(prisonId, false)
       )
-      .filter { !it.isPermanentlyDeactivated() && !it.isActiveAndAllParentsActive() }
```

Comment:
open fun isTemporarilyDeactivated() = !isActiveAndAllParentsActive() && !isPermanentlyDeactivated()

Reply:
Yes it calls isTemporarilyDeactivated